CodeForReview.js

```diff
@@ -0,0 +1,33 @@
+var itemList = [];
```

Comment:
Use const to avoid accidental reassignment
```suggestion
const itemList = [];
```